### PR TITLE
ANALYTICS-3035 | Revamp Docs

### PR DESCRIPTION
**Pull request by travisbauer**
This revamps the Docs to be more consistent and clear, as well as updates the JSON responses/info to match what they currently are.  All RARs docs have been moved to this repo, and this will be the new source for API docs.  Links will be created in the RARs repo. 

A New <internal> tag has been added to tag elements as internal_only, that will hide the info on the api-docs site.

### CODE REVIEW

File: source/includes/_edge_webhook.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-## Edge Webhook
+## CRM Webhooks
+<a name="crm_webhooks"></a>
+
+### Registering the Webhook
 
 Edge has an automation feature that is capable of triggering on “New Lead for New Contact” and “New Lead for Existing Contact” events.  When either of these events occur, this webhook integration will trigger and POST the Lead Event JSON Payload to a URL configured for the webhook.  Create a webhook using the integrations account setting.
```
Comment (clarkbremer) [inline source/includes/_edge_webhook.md:6]: Should e Client Center, not Edge.

File: source/includes/lead_event_feed_service/_overview.md [hunk 2]
```diff
@@ -1,3 +1,4 @@
-## Lead Events [LEFSe] [Calls, Chat, Email, Web Events]
+## Lead Events
+#### [LEFSe] [Calls, Chat, Email, Web Events]
```
Comment (clarkbremer) [inline source/includes/lead_event_feed_service/_overview.md:2]: Not sure why the square brackets.  Or even what this line is for.

File: source/includes/_trouble_shooting_errors.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
-## Troubleshooting
+## Troubleshooting Errors
 
 ### 400 - Bad Request
 
 #### Invalid Grant
```
Comment (clarkbremer) [inline source/includes/_trouble_shooting_errors.md:5]: This renders too small to read.

**Review by clarkbremer — CHANGES_REQUESTED** [review]

File: source/includes/_socialads_campaign_metrics.md [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+|campaign_cycle|Restrict results to a single campaign cycle|
+|interval_size|Use calendar_month or calendar_week to roll up the data points into calendar intervals (default is 1 day per interval)|
+|include_cycles|Set to true or false on whether to include cycle nesting.  Default value is false|
+|<internal> markup_type|Only supported value is 'percentage' </internal>|
+|<internal> markup_value|"cost" fields (spend & budget) will be marked up by this pecentage </internal>|
+
+To specify a date range:
+
+- Specify start_date and end_date.
+- When specifying a **campaign_cycle**, do not use date range params.
```
[the remark this answers is not in the document]
Comment (travisbauer) [reply source/includes/_socialads_campaign_metrics.md:35]: I'd suggest this be another ticket to make all the params consistent.

File: source/includes/lead_event_feed_service/_overview.md (hunk 2 above)
Comment (travisbauer) [reply source/includes/lead_event_feed_service/_overview.md:2]: Not sure either

File: source/includes/_socialads_campaign_metrics.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (travisbauer) [reply source/includes/_socialads_campaign_metrics.md:33]: Jayanthi is making another ticket for this.

File: source/includes/lead_event_feed_service/_overview.md (hunk 2 above)
Comment (travisbauer) [reply source/includes/lead_event_feed_service/_overview.md]: Removed it

**Review by clarkbremer — APPROVED** [review]

**Pull request closed without merging**